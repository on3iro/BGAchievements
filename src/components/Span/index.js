import styled from 'styled-components'

const Span = styled.span`
  display: block;
  margin: ${props => props.error ? '-10px 0 10px 5px' : '0'};
  padding-top: ${props => props.error ? '0' : '5px'}

  color: ${props => props.error ? props.theme.warning : props.theme.outlines};
  font-size: ${props => props.error ? '0.8rem' : 'inherit'};
`

export default Span
