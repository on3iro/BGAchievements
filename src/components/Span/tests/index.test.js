import React from 'react'
import { mount } from 'enzyme'

import Span from '../index'

describe('<Span />', () => {
  it('should render span', () => {
    const renderedWrapper = mount(<Span />)
    expect(renderedWrapper.find('span').length).toMatchSnapshot()
  })
})
