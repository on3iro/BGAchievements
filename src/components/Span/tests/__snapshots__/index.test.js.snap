exports[`<Span /> should render span 1`] = `1`
